buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "92.0.20210803190134"
+    const val VERSION = "92.0.0"
```

Comment:
We don't want to pin `main` to a release version of AC. Similar to Fenix, we want to have release branches that are pinned.